Detectors/ITSMFT/ITS/tracking/src/TrackingInterface.cxx

```diff
@@ -166,7 +166,7 @@ void ITSTrackingInterface::run(framework::ProcessingContext& pc)
   gsl::span<const unsigned char>::iterator pattIt = patterns.begin();
 
   gsl::span<itsmft::ROFRecord> trackROFspan(trackROFvec);
-  mTimeFrame->loadROFrameData(trackROFspan, compClusters, pattIt, mDict, labels);
+  loadROF(mTimeFrame, trackROFspan, compClusters, pattIt, labels);
```

Comment:
Any specific reason not to use the mTimeFrame internally?

Reply:
Since in the ITS3 code we populate the TimeFrame externally with a function in IOUtils, ITS3 needs the pointer.

Reply:
but the ioutils function does not substitute this call no? It should be called inside the ` loadROF` overridden method, which is aware of mTimeFrame